### PR TITLE
Added medium icon in footer and few minor changes

### PR DESCRIPTION
**Pull request by harshs14**
Added medium icon, updated font awesome CDN and updated the icons respectively.
Also updated the link for si logo for google search.

@bvishal8510 @deepanshu-jain1999 @SukhnandanMalhotra please review

### CODE REVIEW

File: Core/urls.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
-from .views import IndexView, RegistrationView, BlogView, BlogDetailView, SaveContactView
+from .views import *
```
Comment (bvishal8510) [inline Core/urls.py]: import all respective names

Comment (harshs14) [reply Core/urls.py]: sir, it is already changed

Comment (harshs14) [reply Core/urls.py]: sir, already done

**Comment by bvishal8510** [comment]
squash the commits into one

**Pull request closed without merging**